projects/feature-activation/0001-feature-activation.md

```diff
@@ -0,0 +1,558 @@
+| `MY_NEW_FEATURE_6` | `DEFINED` | -          | 90%       | 3,500,000    | 3,700,000                 | 3,900,000      | `false`             | v0.52.2       |
+
+The meaning of each of those attributes is explained in the [Criteria configuration] section, except State and Acceptance, which are calculated.
+
+## REST API
+[REST API]: #rest-api
+
+A `GET` endpoint should be provided, mainly to serve the UI described above. It should return a JSON object like the following:
+
+```json
+{
+  "features": [
+    {
+      "name": "MY_NEW_FEATURE_1",
+      "state": "FAILED",
+      "acceptance": 0.8,
+      "threshold": 0.9,
+      "start_height": 3000000,
+      "minimum_activation_height": 3200000,
+      "timeout_height": 3300000,
+      "activate_on_timeout": false,
+      "version": "v0.52.2"
+    }
+  ]
+}
+```
+
+# Reference-level explanation
+[Reference-level explanation]: #reference-level-explanation
+
+In this section, technical details are expanded for what was described above.
+
+## Bit signaling implementation
+[Bit signaling implementation]: #bit-signaling-implementation
+
+Given Hathor's Anatomy of a Transaction RFC [7], it is straightforward to suggest that the bits used for feature flags are in the first (leftmost) byte of the `version` field. From the document:
+
+> The version field takes up 2 bytes. The leftmost byte is reserved for future use and ignored for now. This means that a version with value [0xAA, 0x01] should be interpreted just as `version=1`.
+
+I propose that only the 4 rightmost bits of the leftmost byte are used for feature flag bit signaling. That would still keep 4 bits reserved for future use and ignored, while allowing for up to 4 simultaneous features in different activation processes. If necessary, in the future those 4 leftmost bits could be used to change the interpretation of the 4 rightmost bits. Some examples:
+
+- `[0xA, 0x0, 0x01]`: represents [ignored, no feature support, `version=1`]
+- `[0xC, 0x1, 0x02]`: represents [ignored, feature support for `bit=0`, `version=2`]
+- `[0x0, 0x5, 0x03]`: represents [ignored, feature support for `bit=0` and `bit=2`, `version=3`]
+
+Note: if a miner sends a bit signal for a bit that is not defined as a feature bit, it is going to be ignored.
+
+## Evaluation interval
+
+The evaluation interval, as explained in the [Evaluation interval] section, is defined here to be two weeks (14 days). Considering the 30 seconds average time between blocks of the Hathor network, that interval is equivalent to 40320 blocks. A constant must be defined in `HathorSettings`:
+
+```python
+EVALUATION_INTERVAL = 40320
+```
+
+## Criteria configuration
+[Criteria configuration]: #criteria-configuration
+
+The `Criteria` data class represents the configuration for a certain feature activation criteria:
+
+```python
+class Criteria(BaseModel):
+    name: str
+    bit: int
```

Comment:
Add a comment explaining whether it is LSB or MSB.

Reply:
Added in the "Bit signaling implementation" section, in https://github.com/HathorNetwork/rfcs/pull/51/commits/6a3adcb62aa2b5bc7c0802c9f363f840da6f70a8

Reply:
Thanks!